app/cmd/cwp.go

```diff
@@ -174,6 +175,68 @@ func (o *CWPOptions) Run(cmd *cobra.Command, args []string) (err error) {
 	return
 }
 
+//RunWithoutProcessExits has the same logic with function Run and the difference between them is that RunWithoutProcessExits uses exec.Command() instead of syscall.Exec()
+//which causes the process to exit without executing code after cwp.Run()
+func (o *CWPOptions) RunWithoutProcessExits(cmd *cobra.Command, args []string) (err error) {
```

Comment:
The comment should be like: `// RunWithoutProcessExits xxxx`

You need to add a space.

Reply:
Thx. It's done.